content/blog/2021-02-17-cml-runner-prerelease.md

```diff
@@ -0,0 +1,179 @@
+---
+title:
+  'CML Pre-Release Notes: Automatically Train Models in the Cloud with CML 0.3.0'
+date: 2021-02-22
```

Comment:
date is not consistent with the file name, directory - it might create some problems down the road. Can we fix this please?